contracts/implementation/PoolFactory.sol

```diff
@@ -109,7 +112,7 @@ contract PoolFactory is IPoolFactory, Ownable {
         PoolCommitter poolCommitter = PoolCommitter(
             Clones.cloneDeterministic(poolCommitterBaseAddress, uniquePoolHash)
         );
-        poolCommitter.initialize(address(this));
+        poolCommitter.initialize(address(this), mintingFee, burningFee);
```

Comment:
don't we need to divide by each update interval in the year? this fee seems to be used as is in each commit.

Reply:
better way to phrase the question is minting and burning fee meant to be a flat fee of every mint and burn on top of yearly maintenance fee? 

OR

is the mint and burn fee meant to be an annual fee that we want to divide up into every update interval and extract over the year just like with maintenance fee?

Reply:
The spec given to me explicitly said per commit, so the former :shell: